javascript/config/suites/javascript/regexp

```diff
@@ -0,0 +1,14 @@
++ semmlecode-javascript-queries/DOM/TargetBlank.ql: /Security/CWE/CWE-200
```

Comment:
Why is this query included? It doesn't seem to use regular expressions much.

Reply:
The suite is just here for benchmarking purposes, and the first query is just to pay the cost of computing cached predicates. I'll remove the suite in a rebase.

Reply:
Suite has been removed